### ISSUE TITLE
1.12 fails to load due to OpenAL error

### ISSUE THREAD

**Issue by DestyNova** [Marekkon5/mmlpi#1]
When starting Minecraft 1.12.3 (either vanilla, or with Optifine), things proceed as normal until we try to enter the world, then it crashes with errors like:

```
ALCplaybackAlsa_mixerProc: mmap commit error: Broken pipe
```

It might be to do with the lwjgl version distributed with mmlpi. I switched back to the old method of running Optifine from the command line (i.e. separate to mmlpi) and it's fine.

**Comment by Marekkon5** [comment]
Hello. I was able to run 1.12.2 (1.12.3 doesn't exist) both vanilla and Optifine normally and with sound. You were trying singleplayer or multiplayer? Which OS? Thank you.

**Comment by DestyNova** [comment]
Yes, I was trying to remember the version from memory, figures I'd get it wrong :grinning: 

It's the latest Raspbian running kernel 4.19.66-v7+. I'm not able to access the machine right now to confirm the lsb-release, but it's a fresh install from NOOBS on the same day we tried to run mmlpi, so it's probably Raspbian Buster.

We were trying to start a singleplayer world.

**Comment by Marekkon5** [comment]
Do you renember which Pi were you using and which audio output (HDMI or Analog Jack) were you using? When you'll get access to the machine, could you please delete the old mmlpi, install the new version and try it again. Thank you.

**Comment by yokoyama-flogics** [comment]
Hello,

Thank you for your great software.

I also saw the error "ALCplaybackAlsa_mixerProc: mmap commit error: Broken pipe".
In my case, I could avoid the error by reducing Java VM RAM size to 512 (or 256), from 1024, in the "Play" tab screen.  By this workaround, I could run Minecraft 1.12.2 and also 1.11.2 on my Pi 3 Model B.

I hope it helps.

Regards,
Atsushi
